schemas/stsci.edu/asdf/0.1.0/core/column.yaml

```diff
@@ -0,0 +1,47 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/0.1.0/core/column"
+tag: "tag:stsci.edu:asdf/0.1.0/core/column"
+
+title: >
+  A column in a table.
+
+description: |
+  Each column contains a name and an array of data, and an optional description
+  and unit.
+
+type: object
+properties:
+  name:
+    description: The name of the column
+    type: string
+    pattern: "[A-Za-z_][A-Za-z0-9_]*"
```

Comment:
@embray: But the purpose of the column name is to be able to address it uniquely from some sort of software code, and for that purpose I think an "ASCII token" is still the best for practical reasons.

We can always add a display name later, along with UCD or something like it for more info.